src/doc/rustc/src/platform-support.md

```diff
@@ -300,6 +300,7 @@ target | std | host | notes
 [`i686-unknown-hurd-gnu`](platform-support/hurd.md) | ✓ | ✓ | 32-bit GNU/Hurd [^x86_32-floats-return-ABI]
 [`i686-unknown-netbsd`](platform-support/netbsd.md) | ✓ | ✓ | NetBSD/i386 with SSE2 [^x86_32-floats-return-ABI]
 [`i686-unknown-openbsd`](platform-support/openbsd.md) | ✓ | ✓ | 32-bit OpenBSD [^x86_32-floats-return-ABI]
+`i686-unknown-redox` | ? |  | i686 Redox OS
```

Comment:
I believe every new target **must** have a dedicated platform support page.

Could you add one for this target (or maybe one for all Redox targets) by adding a page under the `platform-support` subdirectory[^1] and link it here as done for other targets.

[^1]: Don't forget to also add the newly created file in the [`SUMMARY.md`](https://github.com/rust-lang/rust/blob/db45105694cc5a63d1c1b39ebb725a024cea2d13/src/doc/rustc/src/SUMMARY.md) file.

Reply:
Added